api/src/main/java/com/datatorrent/api/Operator.java

```diff
@@ -224,6 +224,7 @@ public String toString()
    * Operators must implement this interface if they are interested in being notified as
    * soon as the operator state is checkpointed or committed.
    *
+   * @deprecated Use {@link CheckpointNotificationListener} instead
```

Comment:
We added an additional method and had to introduce the new interface instead of modifying the existing one for backward compatibility.